content/en/hugo-pipes/resource-from-template.md

```diff
@@ -19,7 +19,7 @@ signature: ["resources.ExecuteAsTemplate TARGET_PATH CONTEXT RESOURCE"]
 
 In order to use Hugo Pipes function on an asset file containing Go Template magic the function `resources.ExecuteAsTemplate` must be used.
 
-The function takes three arguments: the target path for the created resource, the template context, and the resource object.
+The function takes three arguments: the target path for the created resource, the template context, and the resource object. The target path is used to cache the result.
```

Comment:
```suggestion
The result is cached using the target path as the cache key.
```

Reply:
I found it useful for it to explain what the first param is. It would be too bad to lose that.

Reply:
Sorry, I meant to replace just the last sentence.